app/js/components/manageApps/ManageApps.jsx

```diff
@@ -289,67 +292,85 @@ export default class ManageApps extends React.Component {
     const { showProgress, uploadStatus} = this.state;
     const progressBar = (
       <div className="progress">
-        <div className="progress-bar" style={{width: uploadStatus + "%"}} />
+        <div className="progress-bar progress-bar-striped" style={{width: uploadStatus + "%"}} />
         <span>{uploadStatus + "%"} Complete</span>
       </div>
     );
 
     return (
       <div>
-        <div className="container-fluid">
-          {this.state.showMsg ? alert : null}
-          {showProgress === true ? progressBar : null}
+        <div className="main-home-page" id="body-wrapper">
+          <div className="row">
+            <div className="col-sm-6">
+              <h2>Add-on Manager</h2>
+            </div>
+            <div className="col-sm-6 manage-settings">
+              <div className="pull-right">
+                <Link to="/manageSettings" className="manage-settings-button">
+                  <i className="glyphicon glyphicon-cog settings-icon" id="icon-btn" />
+                </Link>
+              </div>
+            </div>
+          </div>
+          
+          <div className="home-page-body">
+            <div className="container-fluid">
+              <div id="notification-wrapper">
+                {this.state.showMsg ? alert : null}
```

Comment:
I believe you did not write this piece of code, however I would like to suggest we `short circuit` this line as opposed to rendering `null`. I.e `{this.state.showMsg && alert}`.
This is inline with ReactJS conditional rendering best pratices [HERE](https://facebook.github.io/react/docs/conditional-rendering.html)